### ISSUE TITLE
Error 6002 (Message: copy_file: The system cannot find specified route) 

### ISSUE THREAD

**Issue by Jacketo** [ShikyoKira/Project-New-Reign---Nemesis-Main#296]
Hello! I've been trying to find a fix for my error but I couldn't find one 
Here is the log:
Checking engine version
Engine is up to date
Version Code: 4838-1763

Initializing behavior generation

Mod Checked 1: nemesis
Mod Checked 2: aarm
Mod Checked 3: dscgo

ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately
File: None
Message: copy_file: The system cannot find specified route 

Failed to generate behavior

I deleted all FNIS mods and files, when I attack my character does a T pose (I've installed DAR armory for CGO)

I'm ussing CBE but I'm not using a female character, nor a custom race, do I have to remove the mod anyways? 

And no, even with any mods it keeps telling me that the route is not specified 

Checking engine version
Engine is up to date
Version Code: 9291-4678

Initializing behavior generation

Mod Checked 1: nemesis

ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately
File: None
Message: copy_file: El sistema no puede encontrar la ruta especificada.

Failed to generate behavior

**Comment by ShikyoKira** [comment]
Excluding symbols, do you happen to use any non-latin characters in your PC file system? non-latin characters are characters not from A to Z?

The next thing I would like you to test is try installing Nemesis directly to your skyrim data directory, however, if you aren't experienced enough, don't do this for me. After testing, you can just delete meshes folder and nemesis_engine folder to remove all nemesis generated files. You should backup your meshes folder if it already exist so that after removing it, you can replace the original meshes folder back

I would like you to include your patchlog.txt located in nemesis_engine folder. Thanks for reporting

**Comment by Jacketo** [comment]
Sorry, I'm from argentina so my pc is set on spanish, would that be the problem? 
Do I remove the meshes and nemesis_engine that MO2 generated? or when I install it in Skyrim's data directory?

Also I created an empty mod called "Nemesis Outpost", I saw it in a video but it is supposed optional, could that be a problem too?

So I put it directly in the Skyrim data folder and now the Armoury Animations patch is not available, and the error still appears 

Checking engine version
Engine is up to date
Version Code: 4838-1723

Initializing behavior generation

Mod Checked 1: nemesis
Mod Checked 2: dscgo

ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately
File: None
Message: copy_file: The system could not find the specified route

Failed to generate behavior

[PatchLog.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/4755395/PatchLog.txt)
